packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -68,16 +70,25 @@ export const wrapResolver =
             context.driverConfig = driverConfig;
         }
 
-        if (!context?.jwtConfig) {
-            context.jwtConfig = jwt;
-        }
-
         context.nodes = nodes;
         context.relationships = relationships;
         context.schema = schema;
+        context.plugins = plugins;
```

Comment:
Do we really need to be passing the entire plugin in the context? It seems that we're decoding the JWT and putting in the context below - what more than that do we need?

Reply:
How much destructirng do we want to do here because Im worried about this code that it has the potential to turn into something like this: 

```diff
context.nodes = nodes;
context.relationships = relationships;
context.schema = schema;
context.plugins = plugins;
+ context.authPlugin = plugins.auth;
+ context.autogenerateFunction = plugins.autogenerateFunction ;
+ context.topLevelUnions = topLevelUnions;
+ context.topLevelnterfaces = topLevelUnions;
```

Reply:
For me, it's horses for courses. An autogenerate function, for instance, will need to be called multiple times, so it obviously makes sense for this callback to be in the context. However, JWT decoding is done once, in the wrapper, so it's a complete waste to pass that decode function through in the context. That's where I'm coming from in this particular context (no pun intended).